servers/audio/audio_filter_sw.h

```diff
@@ -73,12 +72,12 @@ class AudioFilterSW {
 	};
 
 private:
-	float cutoff;
-	float resonance;
-	float gain;
-	float sampling_rate;
-	int stages;
-	Mode mode;
+	float cutoff = 5000.0;
```

Comment:
Should these have an `f` affix since they are floats?

Reply:
I can add it, but it is really needed?
Looks that both, with and without f, produce same assembly in godbolt